tremor-script/src/interpreter/imut_expr.rs

```diff
@@ -143,6 +144,21 @@ where
                                 out.push_str(r.encode().as_str());
                             };
                         }
+                        // TODO: The float scenario is different in erlang and rust
+                        // We knowingly excluded float correctness in string interpolation
+                        // as we don't want to over engineer and write own format functions.
+                        // any suggestions are welcome
+                        #[cfg(feature = "erlang-float-testing")]
```

Comment:
Tarpaulin is mean here we got to exclude this code from coverage

```suggestion
                        #[cfg(feature = "erlang-float-testing")]
                        #[cfg(not(tarpaulin_include))]
```